android/app/src/main/cpp/peer_audio_manager.cpp

```
@@ -1,287 +1,386 @@
 }
 
 std::string PeerAudioManager::getMacAddress(int deviceId) {
-    std::lock_guard<std::mutex> lock(peerRegistryMutex);
-    auto it = deviceIdToMac.find(deviceId);
-    return (it != deviceIdToMac.end()) ? it->second : "";
+    std::lock_guard<std::mutex> lock(peerRegistryMutex_);
+    auto it = deviceIdToMac_.find(deviceId);
+    return (it != deviceIdToMac_.end()) ? it->second : std::string{};
+}
+
+bool PeerAudioManager::onVoiceFramePushed(const std::string& macAddress,
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)